SU2_CFD/src/output/filewriter/CParaviewFileWriter.cpp

```diff
@@ -267,19 +268,19 @@ void CParaviewFileWriter::Write_Data(){
   /*--- Need to adjust container location to avoid PointID tag and coords. ---*/
   unsigned short VarCounter = varStart;
 
-  for (unsigned short iField = varStart; iField < fieldnames.size(); iField++) {
+  for (unsigned short iField = varStart; iField < fieldNames.size(); iField++) {
 
-    string fieldname = fieldnames[iField];
+    string fieldname = fieldNames[iField];
 
     fieldname.erase(remove(fieldname.begin(), fieldname.end(), '"'), fieldname.end());
 
```

Comment:
Could be my jet lagged eyes but it looks like this local manipulation of the name is not used?

Reply:
Can you confirm or deny my observation?

Reply:
Its used a couple of lines below (line 287).